entries/a/amazingmarvin.com.json

```diff
@@ -0,0 +1,12 @@
+{
+    "Amazing Marvin": {
+      "domain": "amazingmarvin.com",
+      "tfa": [
+        "totp"
+      ],
+      "documentation": "https://help.amazingmarvin.com/en/articles/5019430-two-factor-authentication",
```

Comment:
```suggestion
      "documentation": "https://help.amazingmarvin.com/en/articles/5019430",
```

Reply:
@phallobst what's the point of shortening the link?

Reply:
@rakleed less bytes with same result.